src/util/native_fs_utils.js

```diff
@@ -229,6 +229,23 @@ async function safe_unlink_posix(fs_context, to_delete_path, to_delete_version_i
     }
 }
 
+/**
+ * unlink_ignore_enoent unlinks a file and if recieved an ENOENT error it'll not fail
+ * @param {nb.NativeFSContext} fs_context 
+ * @param {String} to_delete_path 
+ * @returns {Promise<Void>}
+ */
+async function unlink_ignore_enoent(fs_context, to_delete_path) {
+    dbg.log1('native_fs_utils.unlink_ignore_enoent:', to_delete_path);
+    try {
+        await nb_native().fs.unlink(fs_context, to_delete_path);
+    } catch (err) {
+        dbg.warn(`native_fs_utils.unlink_ignore_enoent unlink error: file path ${to_delete_path} error ${err}`);
```

Comment:
I'm afraid a template string of the object `err` will be printed as `[object Object]`, hence I'm suggesting to print the object as is.
```suggestion
        dbg.warn(`native_fs_utils.unlink_ignore_enoent unlink error: file path ${to_delete_path} error`, err);
```